File: 1470.shuffle-the-array.java
/*
 * @lc app=leetcode id=1470 lang=java
 *
 * [1470] Shuffle the Array
 */

// @lc code=start
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int[] res = new int[2 * n];
        for (int i = 0; i < n; i++) {
            res[2 * i] = nums[i];
            res[2 * i + 1] = nums[n + i];
        }
        return res;
    }
}
// @lc code=end

File: 1470.shuffle-the-array.py
#
# @lc app=leetcode id=1470 lang=python
#
# [1470] Shuffle the Array
#

# @lc code=start
class Solution(object):
    def shuffle(self, nums, n):
        """
        :type nums: List[int]
        :type n: int
        :rtype: List[int]
        """
        res = []
        for i, j in zip(nums[:n],nums[n:]):
            res += [i,j]
        return res
        
# @lc code=end

